### PR TITLE
job scheduling and max retries support added

### PR DESCRIPTION
**Pull request by venkykuberan**
If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [ X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:bug
dataproc : fixed `max_failure_per_hour` not sent in API request for the resource `google_dataproc_job`

```
fixes https://github.com/hashicorp/terraform-provider-google/issues/8319

### CODE REVIEW

**Review by rileykarson — COMMENTED** [review]
While you're here, mind exposing `maxFailuresTotal` as well? We received a request to add it and I figure they're close enough to just get both done.

**Comment by venkykuberan** [comment]
> While you're here, mind exposing `maxFailuresTotal` as well? We received a request to add it and I figure they're close enough to just get both done.

done

File: mmv1/third_party/terraform/resources/resource_dataproc_job.go [hunk 1]
```diff
@@ -153,11 +153,18 @@ func resourceDataprocJob() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"max_failures_per_hour": {
 							Type:         schema.TypeInt,
-							Description:  "Maximum number of times per hour a driver may be restarted as a result of driver terminating with non-zero code before job is reported failed.",
+							Description:  "Maximum number of times per hour a driver may be restarted as a result of driver exiting with non-zero code before job is reported failed.",
 							Required:     true,
 							ForceNew:     true,
 							ValidateFunc: validation.IntAtMost(10),
 						},
+						"max_failures_total": {
```
Comment (rileykarson) [inline mmv1/third_party/terraform/resources/resource_dataproc_job.go:161]: Make sure to update the website as well!

**Review by rileykarson — APPROVED** [review]
LGTM assuming tests pass + the website is updated.

**Pull request merged**